src/controllers/userController.ts

```diff
@@ -290,6 +295,86 @@ const logout = async (req: Request, res: Response) => {
 	}
 };
 
+export const updatePassword = async (req: Request, res: Response) => {
+	try {
+		const { oldPassword, newPassword, confirmPassword } = req.body;
+
+		if (!req.headers.authorization) {
+			return res
+				.status(401)
+				.json(new HttpException("UNAUTHORIZED", "Token is missing"));
+		}
+
+		const token = req.headers.authorization.split(" ")[1];
```

Comment:
Please remove this code, we have middleware which validate if user is authenticated or not

Reply:
I f I remove it, it will bring errors to my codes and test will reduce, I am using the token to get the id of user through decoded, and then I get user, also I am using that to get the user password from the database

Reply:
done